media-gfx/blender/blender-2.78a.ebuild

```diff
@@ -0,0 +1,301 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python3_5 )
+
+inherit check-reqs cmake-utils fdo-mime flag-o-matic gnome2-utils \
+	pax-utils python-single-r1 toolchain-funcs versionator
+
+DESCRIPTION="3D Creation/Animation/Publishing System"
+HOMEPAGE="http://www.blender.org"
+
+SRC_URI="http://download.blender.org/source/${P}.tar.gz"
+
+# Blender can have letters in the version string,
+# so strip of the letter if it exists.
+MY_PV="$(get_version_component_range 1-2)"
+
+SLOT="0"
+LICENSE="|| ( GPL-2 BL )"
+KEYWORDS="~amd64 ~x86"
+IUSE="+boost +bullet +dds +elbeem +game-engine +openexr collada colorio \
+	cuda cycles debug doc ffmpeg fftw headless jack jemalloc jpeg2k libav \
+	llvm man ndof nls openal opencl openimageio openmp opensubdiv openvdb \
+	openvdb-compression player sdl sndfile test tiff valgrind"
+
+# OpenCL and nVidia performance is rubbish with Blender
+# If you have nVidia, use CUDA.
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	player? ( game-engine !headless )
+	cuda? ( cycles !opencl )
+	cycles? ( boost openexr tiff openimageio )
+	colorio? ( boost )
+	openvdb? ( boost )
+	opensubdiv? ( cuda )
+	nls? ( boost )
+	openal? ( boost )
+	opencl? ( cycles )
+	game-engine? ( boost )
+	?? ( ffmpeg libav )"
+
+# Since not using OpenCL with nVidia, depend on ATI binary
+# blobs as Cycles with OpenCL does not work with any open
+# source drivers.
+COMMON_DEPEND="
+	boost? ( >=dev-libs/boost-1.62:=[nls?,threads(+)] )
+	collada? ( >=media-libs/opencollada-1.6.18:= )
+	colorio? ( >=media-libs/opencolorio-1.0.9-r2 )
+	cuda? ( dev-util/nvidia-cuda-toolkit:= )
```

Comment:
you can depend on CUDA 8 stuffs now (and **DON'T** do it via SLOTs! just use normal version specifiers)